sdk/signalr/arm-signalr/CHANGELOG.md

```diff
@@ -1,68 +1,96 @@
 # Release History
 
-## 5.2.1 (Unreleased)
+## 6.0.0-beta.1 (2023-08-31)
 
-### Features Added
+**Features**
 
-### Breaking Changes
+- Added operation group SignalRReplicas
+- Added operation SignalR.listReplicaSkus
+- Added Interface Replica
+- Added Interface ReplicaList
+- Added Interface SignalRListReplicaSkusOptionalParams
+- Added Interface SignalRRegenerateKeyHeaders
+- Added Interface SignalRReplicasCreateOrUpdateOptionalParams
+- Added Interface SignalRReplicasDeleteOptionalParams
+- Added Interface SignalRReplicasGetOptionalParams
+- Added Interface SignalRReplicasListNextOptionalParams
+- Added Interface SignalRReplicasListOptionalParams
+- Added Interface SignalRReplicasRestartHeaders
+- Added Interface SignalRReplicasRestartOptionalParams
+- Added Interface SignalRReplicasUpdateHeaders
+- Added Interface SignalRReplicasUpdateOptionalParams
+- Added Interface SignalRRestartHeaders
+- Added Interface SignalRUpdateHeaders
+- Added Type Alias SignalRListReplicaSkusResponse
+- Added Type Alias SignalRReplicasCreateOrUpdateResponse
+- Added Type Alias SignalRReplicasGetResponse
+- Added Type Alias SignalRReplicasListNextResponse
+- Added Type Alias SignalRReplicasListResponse
+- Added Type Alias SignalRReplicasRestartResponse
+- Added Type Alias SignalRReplicasUpdateResponse
+- Added Type Alias SignalRRestartResponse
+- Interface Resource has a new optional parameter systemData
+- Interface CustomCertificate no longer has parameter systemData
```

Comment:
I remember this has been moved to parent interface, so we can just remove change log like this. as it still can be accessed.